ruby_connectors/lib/ruby_connectors/github_connector/cataloger.rb

```
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module RubyConnectors
+  module GithubConnector
+    class Cataloger
+      def catalog
+        RubyConnectors::Core::ApiCataloger.new(File.join(__dir__, 'schemas')).catalog
```

rubocop

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)